<?xml version="1.0" encoding="UTF-8"?>
<sequence name="com.wso2telco.dep.common.response.unexpectedError.Sequence" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
    <log category="ERROR" level="full">
        <property name="ERROR" value="Unexpected error. Returning error response"/>
    </log>

    <!--Publishing sb related error-->
    <filter xpath="boolean($ctx:sbAction)">
        <call-template target="com.wso2telco.dep.common.main.response.datapublisher.Template">
            <with-param name="msisdn" value="{$ctx:MSISDN}"/>
            <with-param name="direction" value="sb"/>
        </call-template>
    </filter>

    <property action="remove" name="TRANSPORT_HEADERS" scope="axis2"/>
    <property name="httpStatusCode" scope="default" type="STRING" value="500"/>
    <property name="exceptionType" scope="default" type="STRING" value="POLICY_EXCEPTION"/>
    <property name="messageId" scope="default" type="STRING" value="POL0299"/>
    <property name="errorText" scope="default" type="STRING" value="Unexpected Error"/>
    <property name="errorVariable" scope="default" type="STRING" value=""/>
    <sequence key="com.wso2telco.dep.common.response.exceptions.Sequence"/>
</sequence>
